src/core/BufferAttribute.js

```diff
@@ -50,6 +58,18 @@ class BufferAttribute {
 
 	}
 
+	addUpdateRange( offset, count ) {
+
+		this.updateRanges.push( { offset, count } );
+
+	}
```

Comment:
Can we name it `start` instead of `offset` this time?

```js
	addUpdateRange( start, count ) {

		this.updateRanges.push( { start, count } );

	}
```

Reply:
Done!